src/test/regress/citus_tests/config.py

```diff
@@ -190,6 +190,9 @@ def __init__(self, arguments):
         super().__init__(arguments)
         self.worker_amount = 0
         self.is_citus = False
+        self.new_settings = {
```

Comment:
maybe merge this as a separate PR for reference/documentation? Because that is pretty much unrelated to here?

Reply:
agreed. Will open a pr for that soon